Automattic/jetpack
# Social Previews | Add Instagram preview

Completes 1203820933396971-as-1204571006046520/f

## Proposed changes:
* Add Instagram to Social Previews modal

### Other information:


## Jetpack product discussion
pdrWKz-XH-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* In Jetpack, run `jetpack build plugins/jetpack`
* Start writing a new post with a JT site with Social Advanced plan/features enabled
* Add a Featured Image to the post
* Click Jetpack icon on the top of the sidebar
* Under "Social Previews", confirm that you see the Instagram icon
* Click on "Preview"
* Confirm that you see the Instagram preview tab
* Confirm that the preview matches the design - SBgbefzxM6jPVKftsHF7ka-fi-3462_36318
* Please review the related Calypso PR https://github.com/Automattic/wp-calypso/pull/77372


<img width="950" alt="Screenshot 2023-05-25 at 3 09 45 PM" src="https://github.com/Automattic/jetpack/assets/18226415/1b267f65-ef65-41b3-aa34-b6cccfd9fed0">
